frontend/src/app/shared/components/modals/wp-destroy-modal/wp-destroy.modal.html

```diff
@@ -74,10 +72,11 @@
       ></button>
       <button
         class="button button_no-margin -danger spot-action-bar--action"
-        [textContent]="text.confirm"
         [attr.disabled]="busy || blockedDueToUnconfirmedChildren || undefined"
-        (click)="confirmDeletion($event)"
-      ></button>
+        (click)="confirmDeletion($event)">
+        <op-icon icon-classes="button--icon icon-delete"></op-icon>
```

Comment:
Icons can now just be a span:
```suggestion
        <span class="spot-icon spot-icon_delete"></span>
```

Reply:
might still need the `button--icon` class tho

Reply:
I added some overwrite rules to use simple spans of `spot-icon` within the "old" buttons, but this is not merged yet.
See [here](https://github.com/opf/openproject/commit/12ed963f5fca1f3196edc19f456ab6f81680e486#diff-2c53795f153acc4b1f50994a3333ddaf2f6ef82a8ceee970352cc74730af1fb9).

But without this kind of overwrites `spot-icon` spans does not work within the old buttons.